test/shm_size.bats

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function setup() {
+  setup_test
+}
+
+function teardown() {
+  cleanup_test
+}
+
+@test "check /dev/shm is changed" {
+  # Start it with the environment variable set, don't use replace_config
+  CONTAINER_ENABLE_CUSTOM_SHM_SIZE=true start_crio
+
+  # Run base container to ensure it creates at all
+  run crictl runp <(cat "$TESTDATA"/sandbox_config.json | jq '.annotations."io.kubernetes.cri-o.ShmSize" = "16Mi"')
+  echo "$output"
+  [ "$status" -eq 0 ]
+
+  # Run multi-container pod to ensure that they all work together
+  pod_id="$output"
+  run crictl create "$pod_id" "$TESTDATA"/container_sleep.json <(cat "$TESTDATA"/sandbox_config.json | jq '.annotations."io.kubernetes.cri-o.ShmSize" = "16Mi"')
```

Comment:
nit: I don't believe you need the annotations in the container. it doesn't seem like anything processes them

Reply:
Yep, tests still run